src/main/java/duke/Event.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import duke.Task;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
+/**
+ * Event class for events that has start and end date or time.
+ */
+public class Event extends Task{
+    protected String from;
+    //transformed only for printing
+    protected String transfromedFrom;
+    protected String to;
+    protected String transformedTo;
+
+    /**
+     * Constructor for event class
+     * @param description name of the event
+     * @param from start date or time
+     * @param to end date or time
+     */
+    public Event (String description, String from, String to) {
+        //no extra information for todolist
+        super(description);
+        this.from = from;
+        this.to = to;
+        String parsed_from[] = from.split("\\s+");
+        String parsed_to[] = to.split("\\s+");
```

Comment:
Perhaps a better way to instantiate arrays will be something like: 
String[] parsed_from = form.split("\\s+");